final_task/pycalc/pycalc.py

```diff
@@ -0,0 +1,352 @@
+import argparse
+import math
+import string
+import operator
+
+functions = {"abs": operator.abs,
```

Comment:
Aggregation of this dictionary might be automated. Each method in `math` module has it's `name` attribute, which can be used as a key